src/app/core/header/header.component.html

```diff
@@ -1 +1,4 @@
-<p>header works!</p>
+<header class="header-container">
+  <app-logo></app-logo>
+  <app-user-actions></app-user-actions>
```

Comment:
maybe it would be better to name it `AppUserPanel` ? what do you think?

Reply:
Yes, agree, your option sounds better